test/e2e/common/node/runtime.go

```diff
@@ -129,7 +129,8 @@ while true; do sleep 1; done
 
 					ginkgo.By(fmt.Sprintf("Container '%s': should be possible to delete", testContainer.Name))
 					gomega.Expect(terminateContainer.Delete(ctx)).To(gomega.Succeed())
-					gomega.Eventually(ctx, terminateContainer.Present, ContainerStatusRetryTimeout, ContainerStatusPollInterval).Should(gomega.BeFalse())
+					errMsg := fmt.Sprintf("container '%s': should have been terminated by now.", testContainer.Name)
+					gomega.Eventually(ctx, terminateContainer.Present, ContainerStatusRetryTimeout, ContainerStatusPollInterval).Should(gomega.BeFalseBecause(errMsg))
```

Comment:
Can we provide a simple example to illustrate how this change can obtain more info? Will there be more error stacks? Providing simple examples will help everyone understand the benefits of the changes more quickly.

Reply:
It's best practice to use `BeFalseBecause` or `BeTrueBecause` (mentioned in https://onsi.github.io/gomega/). https://github.com/kubernetes/kubernetes/issues/105678 has some examples.

Reply:
thanks!